images/cfw/opt/syslog_shim.py

```diff
@@ -7,7 +7,17 @@
 from logger.tail import TailLog
 import dds
 
-syslog_log = CustomLogger("Syslog parser", "/tmp/x1plus_data.log", 500000, 1)
+# Log setup - Check if SD logging is enabled
+# Start up our own logger (to sd card if SD logging, /tmp/ if not)
+log_path = (
+    "/mnt/sdcard/log/"
```

Comment:
I was going to complain that this should be in `/mnt/sdcard/x1plus/printers/NNNNN/logs/` but it is not a regression over what we have, looking further below